.travis.yml

```diff
@@ -1,63 +1,93 @@
 matrix:
   include:
+    - os: windows
+      language: sh
+      python: 3.6
+      name: Build on Python 3.6.8 in Windows
+      env: PYVER="3.6.8" PYDIR="Python36" INCLUDE="$PWD/cairocomplied/include/" LIB="$PWD/cairocomplied/lib/x64/" CAIRO_VERSION=1.15.12 WHEELSNAMEx32="pycairo-1.19.2-cp36-cp36m-win32.whl" WHEELSNAMEx64="pycairo-1.19.2-cp36-cp36m-win_amd64.whl"
```

Comment:
hm, hardcoding versions isn't nice :( isn't here another way?

Reply:
Idk what is better than this. I have made it so that everywhere in the commands it uses this variable. Please inform me if there is a better one than this.

Reply:
I fixed it in latest commit :) @lazka